modules/oe_translation_cdt/oe_translation_cdt.services.yml

```diff
@@ -0,0 +1,18 @@
+services:
+  oe_translation_cdt.api_client:
```

Comment:
Unfortunately, I couldn't set it to private. It blocks the service replacement in tests. Forcing it to be public would require too many overrides.